nncf/torch/quantization/init_range.py

```diff
@@ -190,15 +247,23 @@ def fwd_hook(module, input_, output):
 
     def _prepare_initialization(self):
         for name, data in self.modules_to_init.items():
-            quantizer_module, init_config = data  # type: BaseQuantizer, RangeInitConfig
+            quantizer_module, init_config, is_weights = data  # type: Tuple[BaseQuantizer, RangeInitConfig, bool]
             num_samples_override = None
             if self.batch_size is not None:
                 num_batches = np.ceil(init_config.num_init_samples / self.batch_size)
                 num_samples_override = num_batches
 
+            if isinstance(quantizer_module, SymmetricQuantizer):
+                mode = 'symmetric'
```

Comment:
This comment is about further improvement, not critical for merging.
IMHO, the existing `QuantizationMode` class is better than strings. Potentially, this mode can be stored in quantized_module.

Reply:
Thanks for the comment, will update!